man/ukify.xml

```diff
@@ -730,46 +718,48 @@ Writing public key for PCR signing to /etc/systemd/tpm2-pcr-public-key-system.pe
     </example>
 
     <example>
-      <title>Multi-Profile PE</title>
-
-      <para>First, create the base UKI:</para>
-      <programlisting>$ ukify build \
-      --linux=/lib/modules/6.0.9-300.fc37.x86_64/vmlinuz \
-      --initrd=/some/path/initramfs-6.0.9-300.fc37.x86_64.img \
-      --cmdline='quiet rw' \
-      --output=base.efi
-      </programlisting>
+      <title>Multi-Profile UKI</title>
 
-      <para>Then, extend the base UKI with information about profile @0:</para>
+      <para>First, create a few profiles:</para>
 
       <programlisting>$ ukify build \
-      --extend=base.efi \
       --profile='TITLE=Base' \
-      --output=base-with-profile-0.efi
+      --output=profile0.efi
       </programlisting>
 
       <para>Add a second profile (@1):</para>
 
       <programlisting>$ ukify build \
-      --extend=base-with-profile-0.efi \
       --profile='TITLE=Boot into Storage Target Mode
 ID=storagetm' \
       --cmdline='quiet rw rd.systemd.unit=stroage-target-mode.target' \
-      --output=base-with-profile-0-1.efi
+      --output=profile1.efi
       </programlisting>
 
       <para>Add a third profile (@2):</para>
 
       <programlisting>$ ukify build \
-      --extend=base-with-profile-0-1.efi \
       --profile='TITLE=Factory Reset
 ID=factory-reset' \
       --cmdline='quiet rw systemd.unit=factory-reset.target' \
-      --output=base-with-profile-0-1-2.efi
+      --output=profile2.efi
+      </programlisting>
+
+      <para>Then, create a UKI and include all the generated profiles:</para>
+
+      <programlisting>$ ukify build \
+      --linux=/lib/modules/6.0.9-300.fc37.x86_64/vmlinuz \
+      --initrd=/some/path/initramfs-6.0.9-300.fc37.x86_64.img \
+      --cmdline='quiet rw' \
```

Comment:
The example is very helpful. I think it could be improved a bit, because the `--cmdline='quiet rw'` might leave people wondering whether or not it could also be in the base profile and/or whether it affects the other profiles. I think adding a small list or table at the end with what profiles with what kernel cmdline are n the resulting UKI would be nice, maybe with another profile named "debug" that's not quiet and ro instead.

Reply:
Let's leave that for a follow up. I'm just fixing the interface here (which I really shouldn't have to do in the first place)